src/workerd/io/compatibility-date.capnp

```diff
@@ -233,11 +233,8 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
 
   nodeJsCompat @21 :Bool
       $compatEnableFlag("nodejs_compat")
-      $experimental;
-  # Experimental, do not use.
+      $compatDisableFlag("no_nodejs_compat")
```

Comment:
Note that this intentionally does not yet add a default enable date.

Reply:
"Yet"? Do we plan to add one eventually or will it always be opt-in?

FWIW the disable flag is meaningless without a default enable date.